Documentation/bootstrapping-guidelines.md

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
I think it would be good to start this document with a description of why you would want to follow this process. Answer the question: "What is the purpose of bootstrapping?"

I would also link such info in the README. If you want to do X, look at document Y.